### ISSUE TITLE
Ciel Nosurge: Ushinawareta Hoshi e Sasagu Uta [PCSG00454]

### ISSUE THREAD

**Issue by Zangetsu38** [Vita3K/compatibility#230]
# Game summary
- Game name: Ciel Nosurge: Ushinawareta Hoshi e Sasagu Uta
- Game serial: PCSG00454
- Game version: 1.04

# Vita3K summary
- Version: v0.1
- Build number: 1075
- Commit hash: https://github.com/vita3k/vita3k/commit/6142543

# Test environment summary
- Tested by: Zangetsu38
- OS: Windows 10
- CPU: AMD Ryzen 2700X
- GPU: NVIDIA GTX 1080 Ti
- RAM: 16 GB DDR4 3600 mhz

# Issues
- Spam OpenGL error: 0x502 on black screen

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/3268614/vita3k.log)

**Comment by iadlast** [comment]
### Vita3k Summary ###
Version: v0.1.4

- Build number: 2094
- Commit hash: https://github.com/Vita3K/Vita3K/commit/b67b7de

### Test Environment Summary ###
- Tested by: iadlast
- OS: Windows 10
- CPU: Intel(R) Core(TM) i7-10750H
- GPU: NVIDIA GeForce GTX 1650 Ti
- RAM: 24 GB DDR4 2933 Mhz

### Issues ###

With the current commit, the game boots normally up to the title screen and selecting New Game plays the opening FMV normally, but afterwards, one of two things happen depending on whether the FMV is allowed to play to the end or whether it is skipped by double tapping the screen.
- Skipping the FMV causes the game to freeze on a neverending "Now Loading" loop with the following log.
[vita3k.log](https://github.com/Vita3K/compatibility/files/8595344/vita3k.log)
- Allowing the FMV to play to the end causes the game to load an impossible scene of a chapter 8 corridor with Greicoff's model acting as if wind were blowing in front of him, and it locks up then and there with the following log.
[vita3k.log](https://github.com/Vita3K/compatibility/files/8595349/vita3k.log)

**Comment by saturnsky** [comment]
# App summary
- App name: Ciel Nosurge Offline
- App serial: PCSG00454
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3253
- Commit hash: https://github.com/vita3k/vita3k/commit/fb470a9f 
- CPU backend: Dynarmic 
- GPU backend: OpenGL 

# Test environment summary
- Tested by: SaturnSky
- OS: Windows 11
- CPU: AMD Ryzen 9 7950X3D
- GPU: AMD Radeon 7900XTX
- RAM: 128 GB

# Issues
- Almost identical to reports in build 2094.
- Fixed audio bug

# Screenshots
- ![image](https://github.com/Vita3K/compatibility/assets/763152/68ca52fd-bceb-4e2c-ab97-5981dc77bbb2)

# Recommended labels

- Ingame -

**Comment by iadlast** [comment]
###App summary###
-App name: Ciel Nosurge Offline
-App serial: PCSG00454
-App version: 1.04
###Vita3K summary###
-Version: v0.2.0
-Build number: 3558
-Commit hash: https://github.com/Vita3K/Vita3K/commit/ed3e7e8a
-CPU backend: Dynarmic
-GPU backend: Vulkan
###Test environment summary###
-Tested by: iadlast
-OS: Windows 11
-CPU: Intel Core i9-12900H
-GPU: NVIDIA GeForce RTX 4060 Laptop GPU
-RAM: 32 GB
###Issues###
-Instant crash on boot with an access violation exception.
[vita3k.log](https://github.com/Vita3K/compatibility/files/14012455/vita3k.log)

**Comment by bookmist** [comment]
Instant crash on boot is fixed by https://github.com/Vita3K/Vita3K/commit/c3e5707f4869e4bb279ffd6cd44d30725874d6c7
Now Ingame-

**Comment by FirionHope** [comment]
Anyone tested it recently? I was looking it up and the only video I saw of it was a few years out of date. The English fan translation is coming out soon, so I was just curious.

**Comment by iadlast** [comment]
No changes at all. The fan translation's patch even includes a section outlining all the problems the emulator has with running the game and why it's impossible to complete it on Vita3k.

**Comment by lone-wolf-akela** [comment]
> No changes at all. The fan translation's patch even includes a section in its Readme outlining all the problems the emulator has with running the game and why it's impossible to complete it on Vita3k.

That's not true. We can now successfully proceeding into the game if we skip the openning FMV (which would lead to a "never-ending 'Now Loading' loop" in older versions of Vita3k as reported above). 
But we still need some kind of camera emulation, otherwise we cannot really enjoy the game.

**Comment by iadlast** [comment]
> > No changes at all. The fan translation's patch even includes a section in its Readme outlining all the problems the emulator has with running the game and why it's impossible to complete it on Vita3k.
> 
> That's not true. We can now successfully proceeding into the game if we skip the openning FMV (which would lead to a "never-ending 'Now Loading' loop" in older versions of Vita3k as reported above). But we still need some kind of camera emulation, otherwise we cannot really enjoy the game.

As far as I know, skipping the opening FMV still causes an endless "Now Loading" loop due to the game failing to load the assets for the scene following it. And that and the camera are far from being the only issue:
-Advancing time either from using the option in the menu or from loading a save crashes the emulator.
-The touch interactions require a multitouch-capable monitor.
-Several of the characters' clothes and hair are glitchy.
-The emulator can crash at random from even loading a save.
-There's a high chance the game will softlock if it needs to load character models for an event scene not set in Ion's Room, the most obvious case of it being the very first scene in the game.
-The credit rolls all have a horrendous case of severe FPS drops that cause them to not finish before the credits song finishes playing and the game returns the player to the map screen or the History menu.

**Comment by lone-wolf-akela** [comment]
> > > No changes at all. The fan translation's patch even includes a section in its Readme outlining all the problems the emulator has with running the game and why it's impossible to complete it on Vita3k.
> > 
> > 
> > That's not true. We can now successfully proceeding into the game if we skip the openning FMV (which would lead to a "never-ending 'Now Loading' loop" in older versions of Vita3k as reported above). But we still need some kind of camera emulation, otherwise we cannot really enjoy the game.
> 
> As far as I know and from the tests I've done myself, skipping the opening FMV still causes an endless "Now Loading" loop due to the game failing to load the assets for the scene following it. And that and the camera are far from being the only issue: -Advancing time either from using the option in the menu or from loading a save crashes the emulator. -The touch interactions require a multitouch-capable monitor. -Several of the characters' clothes and hair are glitchy. -The emulator can crash at random from even loading a save. -There's a high chance the game will softlock if it needs to load character models for an event scene not set in Ion's Room, the most obvious case of it being the very first scene in the game. -The credit rolls all have a horrendous case of severe FPS drops that cause them to not finish before the credits song finishes playing and the game returns the player to the map screen or the History menu.

Well, at least for me, skipping opening FMV does work (not every time, I would say I see a 50% success rate).
And, the touch interactions can also be used with a dualsense controller (with the little touchpad on it).
Other bugs do exist though.

**Comment by RetroGameTalk** [comment]
# Config Tips
![Image](https://github.com/user-attachments/assets/a11ff835-165c-4c69-b1ce-dad09c5f4ab1)

![Image](https://github.com/user-attachments/assets/23f23e0d-ce18-4f10-a6fd-aef11415bd98)

![Image](https://github.com/user-attachments/assets/324358b0-3f11-4927-924b-e131809d9123)

**Comment by iadlast** [comment]
Please note that only the fan translation is fully playable in Vita3k. The base game has none of the modifications that make multitouch and camera support unnecessary.

**Comment by antoniodesousa** [comment]
Thanks for the configuration tips. It really works!
Sadly, the game is unplayable on my system due to severe graphical glitches.

![Image](https://github.com/user-attachments/assets/f6bf85eb-c67c-4e87-b2e9-62875164f391)

OS: Microsoft Windows 11 24H2 Professional
CPU: AMD Ryzen 9 6900HX with Radeon Graphics @ 3.30 GHz
GPU: AMD Radeon RX 6600
RAM: 64 GB

**Comment by Dehela** [comment]
At last I got time to try it (build 4039). The game works really well, except it has a lot of crashes, where most of them I get when I deal with event point restorations (when you get loading satellite thingy in lower-right corner between scene changes). The frequency is so high that I'm not sure how long I will manage to tolerate it.

I tried running on PSTV after that and, well, emulator works and looks just so much better (aside crashes ofc) that I have conflicting feelings. I really wish it worked more stable.
